<?xml version="1.0"?>
<launch>

    <node pkg="sensor_package" type="SensorServer" name="sensor_server" output="screen">
        <rosparam file="$(find sensor_package)/config/sensor_server.yaml" command="load"/>
        <rosparam file="$(find annotation_common)/config/common_parameter.yaml" command="load" />
    </node>

</launch>